From ab263a3e79a36d3e6536adc900b296cdf27f09ae Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Thu, 4 Mar 2021 14:37:06 -0800
Subject: [PATCH] qmldebug: Include limits header

this is needed to solve
qmldebug/qqmlprofilerevent_p.h:314:65: error: 'numeric_limits' is not a member of 'std'

Its exposed bu gcc11

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 src/qmldebug/qqmlprofilerevent_p.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/qmldebug/qqmlprofilerevent_p.h b/src/qmldebug/qqmlprofilerevent_p.h
index a7e37d1964..f778628f8c 100644
--- a/src/qmldebug/qqmlprofilerevent_p.h
+++ b/src/qmldebug/qqmlprofilerevent_p.h
@@ -47,6 +47,7 @@
 #include <QtCore/qvarlengtharray.h>
 #include <QtCore/qmetatype.h>
 
+#include <limits>
 #include <initializer_list>
 #include <type_traits>
 
-- 
2.30.1

